libs/domain/picking/src/services/picking-list-default.service.ts

```diff
@@ -1,27 +1,48 @@
 import { inject } from '@spryker-oryx/di';
-import { BehaviorSubject, map, Observable, shareReplay, switchMap } from 'rxjs';
-import { PickingList, PickingListQualifier } from '../models';
+import {
+  BehaviorSubject,
+  map,
+  Observable,
+  shareReplay,
+  switchMap,
+  tap,
+  using,
+} from 'rxjs';
+import {
+  PickingList,
+  PickingListQualifier,
+  PickingListStatus,
+} from '../models';
 import { PickingListAdapter } from './adapter';
 import { PickingListService } from './picking-list.service';
 
 export class PickingListDefaultService implements PickingListService {
-  protected pickingListsQualifier$ = new BehaviorSubject<PickingListQualifier>(
-    {}
-  );
+  protected pickingListsQualifier$ =
+    new BehaviorSubject<PickingListQualifier | null>(null);
+  protected pickingListsState$ = new BehaviorSubject<PickingList[]>([]);
 
-  protected pickingLists$ = this.pickingListsQualifier$.pipe(
-    switchMap((qualifier) => this.adapter.get(qualifier)),
-    shareReplay({ bufferSize: 1, refCount: true })
+  protected pickingListsLoading$ = this.pickingListsQualifier$.pipe(
+    switchMap((qualifier) =>
+      this.adapter.get(
+        qualifier ?? {
```

Comment:
Not sure regarding this. Why do we need to set default qualifier here? How can we fetch all orders then?

Reply:
Sorry, did the wrong revert. Should be fixed now. I never meant to touch that file.